azure-mgmt-servicefabric/azure/mgmt/servicefabric/__init__.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
```

Comment:
@QingChenmsft Ok, so that's not an Autorest issue, beware of you syntax: `license` and not `licence`